build.sh

```diff
@@ -254,14 +278,22 @@ for i in "$@"
         native)
             __buildnative=true
             ;;
+       x86)
+            __BuildArch=x86
+            ;;
+
         x64)
             __BuildArch=x64
-            __MSBuildBuildArch=x64
             ;;
+
         arm)
             __BuildArch=arm
-            __MSBuildBuildArch=arm
```

Comment:
@nguerrera, it looks like these came over in your initial port of the scripts from coreclr... any reason they're still needed?